projects/1.18-fabric/assets/incantationem/incantationem/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "enchantment.incantationem.bunnys_hop": "兔跳",
+  "enchantment.incantationem.swine_bane": "猪类灾祸",

```

```suggestion
  "enchantment.incantationem.swine_bane": "猪类杀手",
```
原版`节肢杀手`/`Bane of Arthropods`